src/main/com/example008/radek/ExtendedCalculatorV1.java

```diff
@@ -0,0 +1,21 @@
+package main.com.example008.radek;
+
+public class ExtendedCalculatorV1 extends Calculator {
+
+    public boolean isThisNumberOdd(double value) {
+
+        return value % 2 == 0 ? true : false;
```

Comment:
není potřeba psát true false ... nechas jen value % 2 == 0 ... odpoved toho je true nebo false a tu vratis

Reply:
Super ! Už to tam je